packages/@uppy/status-bar/src/StatusBar.jsx

```diff
@@ -1,33 +1,14 @@
 import { UIPlugin } from '@uppy/core'
-import getSpeed from '@uppy/utils/lib/getSpeed'
-import getBytesRemaining from '@uppy/utils/lib/getBytesRemaining'
```

Comment:
If these are unused anymore, maybe you want to remove them?

Reply:
We should definitely remove unused imports, yes, is this the question?

Reply:
I think the `@uppy/utils/lib/getSpeed` and `@uppy/utils/lib/getBytesRemaining` that not used anywhere anymore. Removing them would be technically a breaking change for `@uppy/utils`.